README.md

```diff
@@ -42,11 +42,91 @@ After that, every time you want to run it you may execute the run script: with t
 ./react-bootstrap/run.sh -l
 ```
 
+## Bootstrap options
+
+You can choose between two creation types: Clean and Customized
+
+- ### Clean
+
+Create a clean proyect with the most used libraries of a react proyect
+
+- ### Customized
+
+Choose your preferred libraries of your proyect and create a new react proyect with standarts components, default screens, apis servicies and common utils functions.
+
+###### Components
+- Checkbox
+- InputLabel
+- RadioGroup
+- Routes
+- SearchBar
+- Spinner
+- Suspense 
+- TextArea
+
+###### Screens
+- Login
+- Dashboard
+
+## Bootstrap libraries
+
+|||
+|----|-----------|
+|[react]()| v16.6.3
+
+
+- ### Clean
+
+|Name|Description|
+|----|-----------|
+|[eslint-config-wolox](https://github.com/Wolox/eslint-config-wolox)| Wolox eslint standard rules.
+|[eslint-config-wolox-react](https://github.com/Wolox/eslint-config-wolox-react)| Wolox eslint reactJs rules.
+|[prettier]()| Code formatter.
+|[react-hot-loader](https://www.npmjs.com/package/react-hot-loader)| Allow live reloaded without the loss of state
+|[postcss](https://www.npmjs.com/package/postcss)| Is a tool for transforming styles with JS plugins.
```

Comment:
`Tool for transforming`

Reply:
With capital letter?

Reply:
I mean delete "Is a", just leave `| Tool for transforming styles with JS plugins`